lms/static/sass/base/_layouts.scss

```diff
@@ -81,12 +85,16 @@ body.is-in-course {
 
   // site footer
   .wrapper-footer {
-    margin-top: $baseline;
+    margin-top: ($baseline*2);
     padding-right: 2%;
     padding-left: 2%;
 
-    footer#footer-openedx { // TODO check edX footer when it launches
+    footer#footer-openedx { // shame selector to match existing
       min-width: auto;
     }
   }
+
+  footer#footer-edx-v3 { // shame selector to match existing
+    margin-top: ($baseline*2);
```

Comment:
Also, ick! This should not be in our codebase. Since @nedbat is working on comprehensive theming and next steps involve moving DedX/edx.org specific UI customizations into a formal theme, let's add some comment-based tag we can start searching for elements with. Something like the following maybe?

```scss
// TODO: edX Theme refactor
```